lib/ui/widgets/menu_drawer.dart

```diff
@@ -198,6 +198,18 @@ class MenuDrawer extends StatelessWidget {
               }
             },
           ),
+          ListTile(
+            title: const Text('Groups'),
+            leading: const Icon(Icons.people),
```

Comment:
Maybe it's better to not use the same icon as for member list. Maybe Icons.recent_actors, Icons.person, Icons.contacts or Icons.assignment_ind are good replacements for here of for member list.

Reply:
Try `Icons.groups` (or the outlined version?)

Reply:
try Icons.groups